defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+modified_image_prefix: ''
+modified_append_tag: "{{ lookup('pipe','date +-modified-%Y%m%d%H%M%S') }}"
```

Comment:
My understanding was that this expression is evaluated every time it is referenced, so the timestamp is in danger of changing throughout the run. The set_fact approach avoids that by ensuring it is only evaluated once.